src/main/resources/migreringer/V1.0_0042__oppgavev3_reservasjonstjeneste.sql

```diff
@@ -0,0 +1,46 @@
+alter table saksbehandler add column id BIGINT GENERATED ALWAYS AS IDENTITY NOT NULL;
+alter table saksbehandler add constraint saksbehandler_id_key UNIQUE (id);
+
+create extension if not exists btree_gist;
+create table if not exists RESERVASJON_V3
+(
+    id                          BIGINT GENERATED ALWAYS AS IDENTITY     NOT NULL PRIMARY KEY,
+    reservertAv                 BIGINT                                  NOT NULL,
```

Comment:
Burde vi fortsette lagre saksbehandlere på den måten vi gjør for å gi tilgang i ny versjon? Kan vi heller bruke AD-rollene?

Reply:
Vi har diskutert å lage en annen/bedre modell/tabell for saksbehandlere, men endte med å skyve på problemstillingen, siden vi må forholde oss til den gamle reservasjonsmodellen fortsatt. Min forståelse er at planen er å evt se på dette når gammel reservasjonsløsning er sanert.